org.lamport.tla.toolbox.metainfo.xml

```diff
@@ -33,26 +34,28 @@
     	<category>Development</category>
     	<category>ComputerScience</category>
         <category>Engineering</category>
+        <category>Science</category>
     </categories>
     <description>
-        <p>
-            LA+ is a high-level language for modeling programs and systems--especially concurrent and distributed ones.  It's based on the idea that the best way to describe things precisely is with simple mathematics.  TLA+ and its tools are useful for eliminating fundamental design errors, which are hard to find and expensive to correct in code.
-        </p>
-        <p>
-            The TLA Toolbox is an IDE (integrated development environment) for the TLA+ tools.  You can use it to:
-            <ul>
-            <li>Create and edit your specs, with the locations of parsing errors
-            marked in the modules.</li>
-            <li>Run the PlusCal translator, with the locations of translation
-            errors marked in the PlusCal code.</li>
-            <li>View the pretty-printed versions of your modules.</li>
-            <li>Run the TLC model checker.
+    <p>
+        TLA+ is a high-level language for modeling programs and systems and is useful for eliminating fundamental design errors, which are hard to find and expensive to correct in code.
+    </p>
+    <p>
+        The TLA Toolbox is an IDE (integrated development environment) for the TLA+ tools.  Use it to:</p>
+    <ul>
+        <li>Create and edit your specs, with the locations of parsing errors
+            marked in the modules</li>
+        <li>Run the PlusCal translator, with the locations of translation
```

Comment:
```suggestion
        <li>Run the PlusCal translator, with the locations of translation.
```

Reply:
appstreamcli validate doesn't like final dots in \<li> items.